internal/featuresupport/features_misc.go

```diff
@@ -22,6 +22,10 @@ func (feature *SnoFeature) GetName() string {
 }
 
 func (feature *SnoFeature) getSupportLevel(filters SupportLevelFilters) models.SupportLevel {
+	if filters.HighAvailabilityMode != nil {
+		return ""
```

Comment:
I don't understand why you are doing this, can you explain?

Reply:
Sure, when returning empty string as support level it is not included in the response

Reply:
But I'll try to implement something more clear